oauth2client/service_account.py

```diff
@@ -683,3 +666,44 @@ def _create_token(self, additional_claims=None):
         jwt = crypt.make_signed_jwt(self._signer, payload,
                                     key_id=self._private_key_id)
         return jwt.decode('ascii'), expiry
+
+    def apply(self, headers, access_token=None):
+        """Add the access token as the Authorization header in headers.
+
+        Args:
+            headers: dict, the headers to add the Authorization header to.
+            access_token: An access token to use instead of the credential's
+                current access token.
+        """
+        if not access_token:
+            access_token = self.access_token
+        headers['Authorization'] = 'Bearer ' + access_token
```

Comment:
I will do a follow-up PR soon to normalize all header usage in oauth2client to lowercase bytestrings, cool?